layouts/_default/single.html

```diff
@@ -1,28 +1,27 @@
-{{ define "main" }}
-  <h1>{{ .Title }}</h1>
-
-  {{ with .Description }}
-    <p>{{.}}</p>
-  {{ end }}
-  <aside>
-    {{.TableOfContents}}
-  </aside>
-  <div class="e-content">
-    {{ .Content }}
-
-    <p style="color: #687590;">
-      {{ i18n "lastUpdated" }} {{ .Lastmod.Format "January 2, 2006" }}{{ with .File }} · <a href='https://github.com/mastodon/documentation/tree/main/content/{{ .Lang }}/{{ .Path }}' style="color: #687590;">{{ i18n "improvePage" }}{{ end }}</a>
-
-      {{ if .IsTranslated }}
-        <br />
-
-        {{ i18n "otherTranslations" }}
-
-        {{ range .Translations }}
-          <a href="{{ .RelPermalink }}" style="color: #687590;" hreflang="{{ .Lang }}">{{ .Language.LanguageName }}</a>
-        {{ end }}
-      {{ end }}
-    </p>
-
-  </div>
-{{ end }}
+{{ define "main" }}
+  <h1>{{ .Title }}</h1>  
+  {{ with .Description }}
+    <p>{{.}}</p>
+  {{ end }}
+  <aside>
+    {{.TableOfContents}}
+  </aside>
+  <div class="e-content">
+    {{ .Content }}
+    {{ partial "list.html" . }}

```

Comment:
Including new partial template list.html after generated content. Output is essentially :
` Previous - Page          Next - Page`

To expand on this - "Page" is changed to the page's innerText from the side nav or the page name as Hugo reads it. Links that are alphabetically sorted used Hugo, but had to use JavaScript to make sequential weighted links.